.github/workflows/ci.yml

```diff
@@ -13,6 +13,21 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  test-windows:
+    name: Run tests on Windows
```

Comment:
Maybe we could add to the name that this is with rust tls

Reply:
there are no default features so should be fine afaik

Reply:
done